serving/src/main/java/ai/djl/serving/ModelServer.java

```diff
@@ -525,6 +527,32 @@ String mapModelUrl(Path path) {
             if (engine == null) {
                 return null;
             }
+
+            if ("Python".equals(engine)) {
+                Properties prop = getServingProperties(path);
+                String v = Utils.getenv("TENSOR_PARALLEL_DEGREE", "-1");
+                v = prop.getProperty("option.tensor_parallel_degree", v);
+                int tensorParallelDegree = Integer.parseInt(v);
+                if (tensorParallelDegree > 0) {
+                    String loadOnDevices = configManager.getLoadOnDevices();
+                    if ("*".equals(loadOnDevices)) {
```

Comment:
given the above min and max gpu all set to 1, will this configuration still take effect?